scripts/setup-ecr

```diff
@@ -0,0 +1,130 @@
+mirror_image ${PARALLEL_PULL_FTS_NGINX} "${STANDARD_REPOSITORY}/nginx:parallel-pull-fts"
+mirror_image ${PARALLEL_PULL_FTS_REDIS} "${STANDARD_REPOSITORY}/redis:parallel-pull-fts"
+mirror_image ${PARALLEL_PULL_FTS_REGISTRY} "${STANDARD_REPOSITORY}/registry:parallel-pull-fts"
+
+# Push to ECR in BJS Region
+export AWS_ACCESS_KEY_ID=${ACCESS_KEY}
+export AWS_SECRET_ACCESS_KEY=${SECRET_KEY}
+export AWS_DEFAULT_REGION=${REPLICATE_REGION}
+
+$(aws ecr get-login --region ${AWS_DEFAULT_REGION} --no-include-email)
+
+for image in "amazon-ecs-netkitten" "amazon-ecs-volumes-test" "squid" "awscli" "image-cleanup-test-image1" "image-cleanup-test-image2" "image-cleanup-test-image3" "fluentd" "amazon-ecs-taskmetadata-validator" "amazon-ecs-telemetry-test" "amazon-ecs-containerhealthcheck"; do
+  # Create the repository if it does not exist
+  if ! aws ecr describe-repositories --region ${AWS_DEFAULT_REGION}| grep -q "repositoryName\": \"${image}" ;
+  then
+      aws ecr create-repository --region ${AWS_DEFAULT_REGION} --repository-name "${image}"
+  fi
+  mirror_local_image "amazon/${image}:make" "${REPLICATE_REPOSITORY}/${image}:latest"
+done
+
+
+for image in "executionrole" "busybox" "nginx" "python" "ubuntu" "consul" "debian" "httpd" "mongo" "redis" "registry"; do
+  # Create the repository if it does not exist
+  if ! aws ecr describe-repositories --region ${AWS_DEFAULT_REGION}| grep -q "repositoryName\": \"${image}" ;
+  then
+      aws ecr create-repository --region ${AWS_DEFAULT_REGION} --repository-name "${image}"
+  fi
+done
+
+mirror_local_image "amazon/${ECR_ROLE_IMAGE}:fts" "${REPLICATE_REPOSITORY}/${ECR_ROLE_IMAGE}:fts"
+mirror_local_image ${BUSYBOX_IMAGE} "${REPLICATE_REPOSITORY}/busybox:latest"
+mirror_local_image ${NGINX_IMAGE} "${REPLICATE_REPOSITORY}/nginx:latest"
+mirror_local_image ${PYTHON2_IMAGE} "${REPLICATE_REPOSITORY}/python:2"
+mirror_local_image ${UBUNTU_IMAGE} "${REPLICATE_REPOSITORY}/ubuntu:latest"
+mirror_local_image ${PARALLEL_PULL_FTS_BUSYBOX} "${REPLICATE_REPOSITORY}/busybox:parallel-pull-fts"
+mirror_local_image ${PARALLEL_PULL_FTS_UBUNTU} "${REPLICATE_REPOSITORY}/ubuntu:parallel-pull-fts"
+mirror_local_image ${PARALLEL_PULL_FTS_CONSUL} "${REPLICATE_REPOSITORY}/consul:parallel-pull-fts"
+mirror_local_image ${PARALLEL_PULL_FTS_DEBIAN} "${REPLICATE_REPOSITORY}/debian:parallel-pull-fts"
+mirror_local_image ${PARALLEL_PULL_FTS_HTTPD} "${REPLICATE_REPOSITORY}/httpd:parallel-pull-fts"
+mirror_local_image ${PARALLEL_PULL_FTS_MONGO} "${REPLICATE_REPOSITORY}/mongo:parallel-pull-fts"
+mirror_local_image ${PARALLEL_PULL_FTS_NGINX} "${REPLICATE_REPOSITORY}/nginx:parallel-pull-fts"
+mirror_local_image ${PARALLEL_PULL_FTS_REDIS} "${REPLICATE_REPOSITORY}/redis:parallel-pull-fts"
+mirror_local_image ${PARALLEL_PULL_FTS_REGISTRY} "${REPLICATE_REPOSITORY}/registry:parallel-pull-fts"
+
+# Remove the tag so this image can be deleted successfully in the docker image cleanup integ tests
```

Comment:
We don't explicitly need this here -- its for actually running the tests on the same host